eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -19,9 +20,14 @@ class DbBasedInstaller
     /** @var \Symfony\Component\Console\Output\OutputInterface */
     protected $output;
 
+    /** @var string */
+    protected $baseDataDir;
+
     public function __construct(Connection $db)
     {
         $this->db = $db;
+        // parametrized so other installer implementations can override this
+        $this->baseDataDir = dirname(__DIR__) . '/../../../../data';
```

Comment:
What's the point of using `dirname(__DIR__)` with `/../../../../`? Isn't it the same as: `__DIR__ . '/../../../../../`?

I'm not a fan of positioning relative to current file/dir, because it's not failproof in case of moving file around.

It would be much better to make it relative to root of the bundle, something like:

```php
$installerBundleRoot = $this->container->get('kernel')->locateResource('@EzSystemsPlatformInstallerBundle');
$this->baseDataDir = $installerBundleRoot . . '../../../data';
```

You can always inject bundleRoot from DI with expression:
```yml
arguments:
    # ...
    $installerBundleRoot: "@=service('kernel').locateResource('@EzSystemsPlatformInstallerBundle')"
```

Reply:
> What's the point of using `dirname(__DIR__)` with `/../../../../`? Isn't it the same as: `__DIR__ . '/../../../../../`?

Because I don't like `/../` at all, so less is better ;) But current trends are to replace deterministic function calls with what can be replaced with constant or literal value, so fixed in d97a1e0.

> I'm not a fan of positioning relative to current file/dir, because it's not failproof in case of moving file around.
> It would be much better to make it relative to root of the bundle, something like:
> ```php
> $installerBundleRoot = $this->container->get('kernel')->locateResource('@EzSystemsPlatformInstallerBundle');
> $this->baseDataDir = $installerBundleRoot . . '../../../data';
> ```

It's nice... but for this to be clean, it would require placing `data` in Bundle's `Resources` as bundles tend to be moved around as well.
The next step is to extract PlatformInstallerBundle into a separate package, but keep schema and data files in kernel as they're directly tied to defined Gateways. So I cannot move data there.

Btw. @Nattfarinn: this also brings us to problem we've briefly discussed internally - how to refer to resources in `/data` directory from an external Installer Bundle package? It would need to be a part of a bundle. But which one, since Installer is going to be moved out? It shouldn't be hardcoded `vendor`, because this is configurable by composer settings (I hope we all agree on that - hence change in this PR).